ts/store/reducers/entities/index.ts

```diff
@@ -19,14 +23,16 @@ export type EntitiesState = Readonly<{
   organizations: OrganizationNamesByFiscalCodeState;
   paymentByRptId: PaymentByRptIdState;
   calendarEvents: CalendarEventsState;
+  transactionsRead: ReadTransactionsState;
```

Comment:
Transactions stuff is in the reducer/wallet directory

Reply:
It's a choice made to get persistent data in the app for this functionality. 
In the previous release of the functionality, I noticed an inconsistent behavior after saving data in the redux store.
In particular, closing app and reopening led to incorrect result for what the feature required,  so the choice of using entities section in redux store after consulting with some colleagues.

Reply:
You are right.